src/App.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { Route, Switch, BrowserRouter } from 'react-router-dom';
+import { Route, Switch, BrowserRouter, Link } from 'react-router-dom';
 import './App.css';
 import { AppEnv } from './env';
 
 import Landing from './views/Landing';
 import Login from './views/Login';
 import Register from './views/Register';
 import Resources from './views/Resources';
+import addResources from './views/AddResource'
```

Comment:
Can you make this `AddResources `. Also, there's a small merge conflict here, lmk if you need help resolving it.